llama_index/storage/storage_context.py

```diff
@@ -33,6 +34,7 @@ class StorageContext:
     docstore: BaseDocumentStore
     index_store: BaseIndexStore
     vector_store: VectorStore
+    fs: fsspec.AbstractFileSystem
```

Comment:
curious what is the motivation for storing the `fs` here?

Reply:
if a persist_path is passed to the storage context and it is an s3 url, then a corresponding fs needs to be passed too